src/components/Hero/index.tsx

```diff
@@ -0,0 +1,52 @@
+import Image from 'next/image';
+import Team from '../../public/team.jpg';
+import Button from '../Button/index';
```

Comment:
Instead of use `../../` to access the src folder you can use the `@` symbol
You do not need the `/index` for the Button